typescript/sdk/src/token/CosmosNativeWarpModule.ts

```diff
@@ -402,9 +402,9 @@ export class CosmosNativeWarpModule extends HyperlaneModule<
         addresses: {
           ...this.args.addresses,
           mailbox: expectedConfig.mailbox,
-          deployedIsm: (
-            actualConfig.interchainSecurityModule as DerivedIsmConfig
-          ).address,
+          deployedIsm:
+            (actualConfig.interchainSecurityModule as DerivedIsmConfig)
+              ?.address ?? '',
```

Comment:
Isn't this a bit dangerous? can you explain why it is set to an empty string as fallback? wouldn't it be better to explicitly deploy if no ism is set on the current config?

Reply:
Had to do it because in the edge case where there is currently no custom ism for a warp route deployed and now there should be created one in warp apply it would fail otherwise. It is handled here: https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/feat/cosmos-warp-apply/typescript/sdk/src/ism/CosmosNativeIsmModule.ts#L94